frontend/src/components/Layout.tsx

```diff
@@ -168,6 +168,17 @@ export default function PersistentDrawerLeft(props) {
 						</ListItemButton>
 					</ListItem>
 				</List>
+				<Divider/>
+				<List>
+					<ListItem key={"newmetadata"} disablePadding>
+						<ListItemButton component={RouterLink} to="/new-metadata-definition">
+							<ListItemIcon>
+								<AddBox/>
```

Comment:
I would change to another ICON just to differentiate with the new dataset icon. Maybe this: `import CreateIcon from '@material-ui/icons/Create';`?

Reply:
Updated
<img width="1624" alt="Screen Shot 2022-10-27 at 1 19 01 AM" src="https://user-images.githubusercontent.com/28983148/198206283-1cd9ced2-28c0-4a93-a291-20199aaa1808.png">